.github/workflows/continuous-integration-workflow.yml

```
@@ -165,11 +165,29 @@ jobs:
           path: playwright-report
           retention-days: 30
           if-no-files-found: error
+      - name: Archive code coverage results
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)